iati/tests/test_validator.py

```diff
@@ -415,12 +410,37 @@ def test_basic_validation_valid(self, schema_basic):
         assert iati.validator.is_iati_xml(data, schema_basic)
         assert iati.validator.is_valid(data, schema_basic)
 
-    def test_basic_validation_should_pass_from_ssot(self, iati_dataset_valid_from_ssot, schema_basic):
+    @pytest.mark.parametrize('file_path', iati.tests.resources.get_test_data_paths_in_folder('ssot-activity-xml-pass') + iati.tests.resources.get_test_data_paths_in_folder('ssot-org-xml-pass'))
+    def test_basic_validation_should_pass_from_ssot(self, file_path, schema_basic, schema_basic_org):
         """Perform check to see whether a parameter is valid IATI XML.
 
         The parameter is valid IATI XML. It is sourced from the SSOT.
         """
-        assert iati.validator.is_iati_xml(iati_dataset_valid_from_ssot, schema_basic)
+        dataset = iati.utilities.load_as_dataset(file_path)
+
+        if 'ssot-activity-xml-pass' in file_path:
+            schema = schema_basic
+        else:
+            schema = schema_basic_org
+
+        assert iati.validator.is_xml(dataset.xml_str)
+        assert iati.validator.is_iati_xml(dataset, schema)
+
+    @pytest.mark.parametrize('file_path', iati.tests.resources.get_test_data_paths_in_folder('ssot-activity-xml-fail') + iati.tests.resources.get_test_data_paths_in_folder('ssot-org-xml-fail'))
+    def test_basic_validation_should_fail_from_ssot(self, file_path, schema_basic, schema_basic_org):
+        """Perform check to see whether a parameter is valid IATI XML.
+
+        The parameter is valid IATI XML. It is sourced from the SSOT.
+        """
```

Comment:
This shouldn't have passed the linters. Did you lint this locally?

Reply:
linting is all passing

Reply:
http://www.pydocstyle.org/en/2.1.1/error_codes.html indicates `D410`requires a blank line after a section. This docstring has no sections, so all is well!